### PR TITLE
Éric comments

### PR DESCRIPTION
**Pull request by paulehoffman**

### CODE REVIEW

File: draft-ietf-cbor-7049bis.md [hunk 1]
```diff
@@ -1014,7 +1014,7 @@ format is invalid.
 ### Epoch-based Date/Time {#epochdatetimesect}
 
 Tag number 1 contains a numerical value counting the number of seconds
-from 1970-01-01T00:00Z in UTC time to the represented point in civil
+from 1970-01-01T00:00Z in UTC time to the represented point in civil (clock)
 time.
```
Comment (cabo) [inline draft-ietf-cbor-7049bis.md:1018]: That change does not work for me.  Civil time is a term of art, and all relevant civil time scales are now based on UTC.
Wikipedia has the nice definition of civil time: "statutory time scales designated by civilian authorities", but also cops out at "local time indicated by clocks".
So if we want to explain "civil time", we need to include the whole thing (or, point to a source, which might even be WIkipedia).

File: draft-ietf-cbor-7049bis.md [hunk 2]
```diff
@@ -1911,7 +1912,7 @@ data items to be encoded without harm.
 # Converting Data between CBOR and JSON
 
 This section gives non-normative advice about converting between CBOR
-and JSON. Implementations of converters are free to use whichever
+and JSON. Implementations of converters MAY use whichever
```
Comment (cabo) [inline draft-ietf-cbor-7049bis.md:1915]: RFC 2119:
5. MAY   This word, or the adjective "OPTIONAL", mean that an item is truly optional. 
Well, OK, it is optional to use the advice here, but that is a bit of RFC 2119 abuse.

**Review by cabo — CHANGES_REQUESTED** [review]

File: draft-ietf-cbor-7049bis.md (hunk 2 above)
Comment (paulehoffman) [reply draft-ietf-cbor-7049bis.md:1915]: Disagree.

File: éric-comments [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+> Is there a reason why "specifically, tag number 25 and tag number 29" have no
+> reference to a RFC ? The reader would benefit from some short description. This
+> oddity was also mentioned by Eve in her review, so, I strongly suggest to
+> address the issue.
+
+As you can see at <https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml#tags>,
+these are not RFCs, they are to a personal website. Instead of creating a link in an
+RFC that could go stale, it is better to expect a developer who cares about the tags
+to go to the IANA registry.
+
```
[the remark this answers is not in the document]
Comment (paulehoffman) [reply éric-comments:84]: I do not see a reason to add another pointer to the IANA registristry in this section. Either an implementer knows about it, or they aren't really implementing those tags.

File: draft-ietf-cbor-7049bis.md [hunk 4]
```diff
@@ -393,7 +393,7 @@ in the environment).  The ability to provide generic encoders and
 decoders is an explicit design goal of CBOR; however many applications
 will provide their own application-specific encoders and/or decoders.
 
-In the basic (un-extended) generic data model, a data item is one of:
+In the basic (un-extended) generic data model defined in {{majortypes}}, a data item is one of:
```
[the remark this answers is not in the document]
Comment (paulehoffman) [reply draft-ietf-cbor-7049bis.md:397]: Done

File: draft-ietf-cbor-7049bis.md [hunk 5]
```diff
@@ -504,7 +504,7 @@ Less than 24:
   If the major type is 0, 1, or 6, the encoded item is not
   well-formed.  For major types 2 to 5, the item's length is
   indefinite, and for major type 7, the byte does not consitute a data
```
[the remark this answers is not in the document]
Comment (paulehoffman) [reply draft-ietf-cbor-7049bis.md:506]: Fixed

File: draft-ietf-cbor-7049bis.md [hunk 6]
```diff
@@ -569,11 +569,11 @@ Major type 3:
   well-formed but invalid ({{terminology}}). This type is provided for
   systems that need to interpret or display human-readable text, and
   allows the differentiation between unstructured bytes and text that
-  has a specified repertoire and encoding.  In contrast to formats
+  has a specified repertoire (Unicode) and encoding (UTF-8).  In contrast to formats
```
[the remark this answers is not in the document]
Comment (paulehoffman) [reply draft-ietf-cbor-7049bis.md:572]: Sure. But do remember that I'm co-author of the i18n terminology document. :-)

File: draft-ietf-cbor-7049bis.md (hunk 1 above)
Comment (paulehoffman) [reply draft-ietf-cbor-7049bis.md:1018]: I'll revert this. Geez, people get twitchy talking about dates and times...

**Pull request merged**